src/ShapeCrawler/Shapes/IPath.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace ShapeCrawler.Shapes;
```

Comment:
CI Pipeline is failed due to "SC0003" issue: public interfaces should have `ShapeCrawler` namespace. Please rename `ShapeCrawler.Shapes` on `ShapeCrawler`.

Use "Release" configuration to see the code style issues on you local environment.

Reply:
OK, no errors found but some warnings received, should I ignore it?

Reply:
Yes. It can be ignored.